### DISCUSSION TITLE
Does this library use a vulnerable version of log4j?

### DISCUSSION THREAD

**Discussion by AnonymousQuestions** [AbsaOSS/ABRiS#259] [Q&A] [answered]
Hi, I was wondering if this library is using a version of log4j that could be exploited with the well-known vulnerability or if it’s using a version that is safe.

**Comment by cerveada** [comment, marked as answer]
**No**

Abris is using logging from Spark. Spark uses older log4j (1.2.17) that is not affected by CVE-2021-44228.

**Reply by AnonymousQuestions** [reply]
Thanks!
